docs/common_tasks.rst

```diff
@@ -51,6 +51,42 @@ Running the tests
 
      (venv)vagrant@precise32:~/treeherder$ flake8
 
+
+Look up credentials for a project
+---------------------------------
+
+To submit data to an existing project, you need corresponding OAuth credentials, even if you're submitting to your local server.
+
+Prerequisite: you must have previously exported credentials for all data source projects. This is part of the usual Treeherder setup process.
+
+  .. code-block:: bash
+
+      > ./manage.py export_project_credentials
```

Comment:
The usual practice here is to specify the prompt so it's clear these commands need to be run inside the virtual environment. In this case you'd write:

(venv)vagrant@precise32:~/treeherder$ ./manage.py export_project_credentials